proxy/src/graphql/schema.rs

```diff
@@ -129,7 +129,18 @@ impl Mutation {
         id: juniper::ID,
     ) -> Result<registry::Transaction, error::Error> {
         // TODO(xla): Get keypair from persistent storage.
-        let fake_pair = ed25519::Pair::from_legacy_string("//Alice", None);
+        let fake_pair =
+            ed25519::Pair::from_legacy_string(&format!("//{}", handle.to_string()), None);
+
+        use radicle_registry_client::CryptoPair;
+        // Give new account some dough so we can perform transactions.
```

Comment:
```suggestion
        // The new account needs funds to perform transactions.
```

"dough" sounds cool but such language is unhelpful here, imo. Also, I suggest explaining _why_ we are doing it, the _what_ the code tells already.

Reply:
It's a temporary measure and I allowed myself some fun with it. Will add a comment to remove the method alltogether.